projects/canopy/src/lib/details/details-panel-heading/details-panel-heading.component.ts

```diff
@@ -9,7 +9,14 @@ import {
 } from '@angular/core';
 import { NgIf, NgSwitch, NgSwitchCase } from '@angular/common';
 
-import { lgIconChevronDown } from '../../icon';
+import {
+  lgIconCheckmarkSpotFill,
+  lgIconChevronDown,
```

Comment:
you've imported this icon, but not registered it?

Reply:
@Xinchro Good spot! I've added that now, and combined the LgIconComponent import which I realised was on a separate line